src/controllers/scripting/legacy/controllerscriptenginelegacy.cpp

```diff
@@ -185,73 +149,42 @@ bool ControllerScriptEngineLegacy::callShutdownFunction() {
 bool ControllerScriptEngineLegacy::callInitFunction() {
     // m_pController is nullptr in tests.
     const auto controllerName = m_pController ? m_pController->getName() : QString{};
+    const auto args = QJSValueList{
+            controllerName,
```

Comment:
implicit copy instead of move.
```suggestion
    const auto args = QJSValueList{
             m_pController ? m_pController->getName() : QString{},
```